### DISCUSSION TITLE
How to style custom fieldtypes on frontend

### DISCUSSION THREAD

**Discussion by earthwytch** [statamic/cms#8614] [Q&A]
Hi everyone,

I have created a basic fieldtype for a frontend form and successfully added it in the blueprint and it is showing in the frontend form. It is currently getting its styling from `resources/views/vendor/statamic/forms/fields/default.antlers.html`. 

How/where can I create a custom view for the new fieldtype if my fieldtype was called SpecialThing in `app/Fieldtypes/SpecialThing.php`.

I have tried adding:

`resources/views/forms/fields/special-thing.antlers.html`
`resources/views/forms/fields/specialthing.antlers.html`
`resources/views/forms/fields/special_thing.antlers.html`

and:

`resources/views/vendor/statamic/forms/fields/special-thing.antlers.html`
`resources/views/vendor/statamic/forms/fields/special_thing.antlers.html`
`resources/views/vendor/statamic/forms/fields/specialthing.antlers.html`

and even:

`resources/views/iextend/forms/fields/special-thing.antlers.html`
`resources/views/iextend/forms/fields/special_thing.antlers.html`
`resources/views/iextend/forms/fields/specialthing.antlers.html`

But it still only picks up the default view file. Can anybody help save me from spamming this file in every folder?

**Comment by edalzell** [comment]
So you're using the `fields` tag to loop through the form fields?
